message VoltageReading {
   required float voltage = 1;
   required fixed64 timestamp_millis_unix_epoch = 2;
}
